pkg/pillar/types/locationconsts.go

```diff
@@ -75,6 +75,8 @@ const (
 	ITokenFile = "/run/eve.integrity_token"
 	//EveVersionFile contains the running version of EVE
 	EveVersionFile = "/run/eve-release"
+	//EveVirtTypeFile contains the virtualization type, ie kvm, xen or kubevirt
+	EveVirtTypeFile = "/run/eve-virt-type"
```

Comment:
Please modify this to /run/eve-hv-type.   I am going to update my PR https://github.com/lf-edge/eve/pull/3231 to reflect that change.

Reply:
done.